libsolidity/interface/StandardCompiler.cpp

```diff
@@ -427,8 +428,7 @@ boost::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompile
 	if (auto result = checkRootKeys(_input))
 		return *result;
 
-	if (_input["language"] != "Solidity")
-		return formatFatalError("JSONError", "Only \"Solidity\" is supported as a language.");
+	ret.language = _input["language"].asString();
```

Comment:
Could add a check here, similar to the `check*` functions, that only `Solidity` and `Yul` are allowed.

Reply:
It is in StandardCompiler.cpp:931